COMPATIBILITY.md

```diff
@@ -116,6 +116,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [ ] `ownership` : [JENKINS-32353](https://issues.jenkins-ci.org/browse/JENKINS-32353)
 - [ ] `job-restrictions`: [JENKINS-32355](https://issues.jenkins-ci.org/browse/JENKINS-32355)
 - [X] `buildtriggerbadge`: supported as of 2.2
+- [X] `build-monitor`: supported as of 1.6+build.159 
```

Comment:
Actually [it seems to be `build-monitor-plugin`](https://github.com/jan-molak/jenkins-build-monitor-plugin/blob/14ad61aa52c78158861706f50d51fdc9bd9b7a9c/pom.xml#L11), not `build-monitor` (too late to fix).